modules/install-ibm-cloud-preparing-the-provisioner-node.adoc

```diff
@@ -0,0 +1,225 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_cloud/install-ibm-cloud-installing-on-ibm-cloud.adoc
+
+[id="preparing-the-provisioner-node-for-openshift-install-on-ibm-cloud_{context}"]
+= Preparing the provisioner node for {product-title} installation on IBM Cloud
+
+This procedure is a substitute for the xref:../installing_bare_metal_ipi/ipi-install-installation-workflow.html#preparing-the-provisioner-node-for-openshift-install_ipi-install-installation-workflow[Preparing the provisioner node for OpenShift Container Platform installation] procedure in the "Setting up the environment for an OpenShift installation" section of the "Deploying installer-provisioned clusters on bare metal" documentation. When creating the provisioner node in the IBM Cloud, select the operating system. Then, perform this procedure. Once you complete this procedure, retrieve and extract the {product-title} installer and configure the public subnet before proceeding with configuring the `install-config.yaml` file.
```

Comment:
maybe just me reading this but I was unsure of the need for a full stop here. 
When creating the provisioner node in the IBM Cloud, select the operating system, then perform this procedure. 
Maybe I am missing something as I have not carried this out.